specificationRepositoryConfiguration.json

```diff
@@ -19,6 +19,11 @@
       "integrationRepository": "AzureSDKAutomation/azure-sdk-for-net",
       "mainRepository": "Azure/azure-sdk-for-net"
     },
+    "azure-sdk-for-net-track2": {
```

Comment:
Sorry I know this PR is merged already but before we start using this in a bunch of places can we rename it? I don't like using the terminology of "track2" in these artifacts as that is a code name that doesn't really mean anything. It would be better if we called it something that was more connected to the changes. "azure-sdk-for-net-v3-generator" or just "azure-sdk-for-net-generator".

Reply:
It's up to your team to choose a name. You only need to update this config file to make it work. I'm OK for this change.

Reply:
@ShivangiReja as part of your work could you please update this name before starting to do the configurations.